From 11d1eb3acd0746f3edcd3b96faad282271257f97 Mon Sep 17 00:00:00 2001
From: Joe Gordon <jogo@cloudscaling.com>
Date: Wed, 2 Nov 2011 16:21:21 -0700
Subject: [PATCH] fixes bug 883233.

fix typo in scheduler/driver.py assert_compute_node_has_enough_memory.

(cherry picked from commit fbe692f9d727ff1f96b960794f637f26a85a61cb)

Change-Id: Idce87927a35b5986c970e6293428b3a5fddd00e4
---
 Authors                  |    1 +
 nova/scheduler/driver.py |    3 ++-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/Authors b/Authors
index cadfb26..cedd199 100644
--- a/Authors
+++ b/Authors
@@ -51,6 +51,7 @@ Jason Koelker <jason@koelker.net>
 Jay Pipes <jaypipes@gmail.com>
 Jesse Andrews <anotherjesse@gmail.com>
 Jimmy Bergman <jimmy@sigint.se>
+Joe Gordon <jogo@cloudscaling.com>
 Joe Heck <heckj@mac.com>
 Joel Moore <joelbm24@gmail.com>
 Johannes Erdfelt <johannes.erdfelt@rackspace.com>
diff --git a/nova/scheduler/driver.py b/nova/scheduler/driver.py
index 22f4e14..43e1382 100644
--- a/nova/scheduler/driver.py
+++ b/nova/scheduler/driver.py
@@ -291,7 +291,8 @@ class Scheduler(object):
         if avail <= mem_inst:
             instance_id = ec2utils.id_to_ec2_id(instance_ref['id'])
             reason = _("Unable to migrate %(instance_id)s to %(dest)s: "
-                       "Lack of disk(host:%(avail)s <= instance:%(mem_inst)s)")
+                       "Lack of memory(host:%(avail)s <= "
+                       "instance:%(mem_inst)s)")
             raise exception.MigrationError(reason=reason % locals())
 
     def assert_compute_node_has_enough_disk(self, context,
-- 
1.7.6.5

